e2e/vertical-full-page-map.spec.ts

```diff
@@ -17,27 +17,41 @@ test.describe('full page map test suite', () => {
     await page.getByPlaceholder('Search for locations').fill('virginia');
     await page.getByPlaceholder('Search for locations').press('Enter');
     await page.getByRole('button', { name: 'Result number 5' }).click();
-    const locator = page.locator('#js-answersVerticalResults div').nth(134);
-    await expect(locator).toHaveClass(/pinFocused/);
+    const results = page.locator('#js-answersVerticalResults div');
+    const count = await results.count();
+    let hasPinFocused = false;
+
+    for (var i = 0; i < count; i++) {
+      const currResult = await results.nth(i);
+      const classes = await currResult.evaluate(node => Array.from(node.classList));
+
+      if (classes.includes('yxt-Card--pinFocused')) {
+        hasPinFocused = true;
+        break;
+      }
+    }
+
+    expect(hasPinFocused).toBe(true);
   });
 
   test('search when map moves works', async ({ page }) => {
-    const searchLogo = '#js-yext-submit';
-    await page.mouse.move(600, 300);
-    await page.mouse.down();
-    await page.mouse.move(1200, 450, {steps: 5});
-    await page.mouse.up();
-    await page.waitForSelector(searchLogo, { state: 'detached' });
-    const detachedSearchLogo = await page.$(searchLogo);
-    expect(detachedSearchLogo).toBeFalsy();
+    await page.getByPlaceholder('Search for locations').fill('virginia');
+    await page.getByPlaceholder('Search for locations').press('Enter');
```

Comment:
what's the reason for conducting a search at the start of this test? couldn't the `waitForResponse` below be matching this search instead of the one triggered from moving the map?